vcr/concept/phonetics_test.go

```diff
@@ -22,7 +22,7 @@ package concept
 import (
 	"testing"
 
-	"github.com/nuts-foundation/go-leia"
+	"github.com/nuts-foundation/go-leia/v2"
```

Comment:
why the v2 package?

Reply:
Not backwards compatible with v1, then go dictates to add the version number after the package